composer.json

```diff
@@ -32,18 +32,17 @@
         "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "fig/http-message-util": "^1.1",
         "laminas/laminas-escaper": "^2.10.0",
-        "psr/http-message": "^1.0",
-        "psr/http-server-middleware": "^1.0"
+        "psr/http-message": "^1.0 || ^2.0",
+        "psr/http-server-middleware": "^1.0.2"
     },
     "require-dev": {
         "laminas/laminas-coding-standard": "~2.5.0",
-        "laminas/laminas-diactoros": "^2.22",
+        "laminas/laminas-diactoros": "^2.25 || ^3.0",
```

Comment:
My main reason to keep 1.0 support was because I'm not 100% certain which of our components depend explicitly on Diactoros, and this would allow usage with v2 or v3 of that library until those are updated.

I think in 3.11 we could drop 1.0 support.